mlos_bench/mlos_bench/storage/__init__.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Interfaces to the storage backends for OS Autotune.
+"""
+
+from mlos_bench.storage.base_storage import Storage
+from mlos_bench.storage.sql_storage import SqlStorage
+
+__all__ = [
+    'Storage',
+    'SqlStorage',
```

Comment:
Might make sense to skip loading the SQL one by default in case they aren't using that.  Since it's the only option for now, call it future work perhaps?

Reply:
you're right, we don't use it anywhere directly. Will remove the export.

Reply:
Actually, no. I've just rolled back this change. We need this export to instantiate `SqlStorage` from JSON config - the same way we do it for `Environment` and `Optimizer` classes.

Reply:
I don't follow.  Can't we "just" specify the full module path in the json files?

Reply:
Some reasons I wonder:
- There's generally been a principle that when this package is deployed, it should be loadable without requiring _all_ of the various supported dependencies.  For instance, in the `mlos_core` pacakge, we try to not require you to have `emukit` installed if you aren't using it.  Similarly, here if we aren't using a SQL backend, it'd be nice not to require the sqlalchemy package to be installed.  This is something we could probably fix later once we have multiple backends, but I suspect we'd forget.
- In the course of doing a bunch of the mypy work I ran across a lot of circular dependency issues with loading from a module higher in the stack as opposed to something lower down and more specific.  Maybe that's something we can handle just internally moreso, but in the spirit of least privileges/dependencies, it seemed worth considering.

Anyways, not a strong preference, just something to consider.

Reply:
ok I'm convinced 😄 I've re-removed the export of `SqlStorage` and changed our JSONC configs so that they import this class directly as `mlos_bench.storage.sql_storage.SqlStorage`